app/Http/Controllers/ReviewController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Movie;
+use App\Models\Review;
+use App\Http\Requests\ReviewRequest;
+
+class ReviewController extends Controller
+{
+    public function index()
+    {
+        $reviews = Review::with(['movie'])
+            ->latest()
+            ->paginate(20);
+
+        return view('reviews.index', [
+            'reviews' => $reviews
+        ]);
+    }
+
+    public function show(Review $review)
+    {
+        $review->with([
+            'movie'
+        ])->get();
+
+        return view('reviews.show', [
+            'review' => $review
+        ]);
+    }
+
+    public function create(Movie $movie)
+    {
+        return view('reviews.create', [
+            'movie' => $movie
+        ]);
+    }
+
+    public function store(ReviewRequest $request)
+    {
+        Review::create($request->validated());
+
+        return redirect()->route('reviews.index')
+                ->with(['message' => 'Your review has been added.']);
```

Comment:
Optional - There is another way to do this to :D [Redirecting With Flashed Session Data]( https://laravel.com/docs/8.x/redirects#redirecting-with-flashed-session-data)

Maybe this makes it a tiny bit more clear on the controller and view ;D.